pkg/operator/reporting/templates_test.go

```diff
@@ -84,7 +97,7 @@ func TestRenderQuery(t *testing.T) {
 			name: "invalid report query with invalid templating (missing Inputs field) returns error",
 			reportTemplate: &ReportQueryTemplateContext{
 				ReportQuery: newTestReportQuery(testReportQuery, testNamespace, "SELECT * FROM {| dataSourceTableName .Report.Inputs. |}", []metering.ReportQueryInputDefinition{
-					newTestInputDefinition("ReportDataSource"),
+					newTestInputDefinition("", ""),
```

Comment:
Why an empty type here? I think this should be `newTestInputDefinition("", "ReportDataSource")`?

Reply:
Actually, again, why are any input definitions not specifying both? I think we should be testing where the ReportQuery itself is validated and has a name/type set in all cases.